gfsphysics/CMakeLists.txt

```diff
@@ -15,174 +15,6 @@ set(CCPP_SOURCES
     physics/namelist_soilveg.f
     physics/set_soilveg.f
     physics/noahmp_tables.f90
```

Comment:
can these three files be moved to ccpp? why do they need to exist in gfsphysics directory? what's the purpose of this directory (library)?

Reply:
Do you mean temporarily until they are removed entirely? These files do exist in the ccpp-physics repository already. The only reason why they also exist in gfsphysics is that `FV3GFS_io.F90` does some initialization of the LSM tables. The existence of the IPD didn't allow us to do this entirely in CCPP.

Moving this to CCPP and deleting those three files will be step 2 of the IPD removal process, to come next. In so far I don't think it makes sense to move them for just one commit. Note that this step 2 will change the answer for 32BIT model runs, because the LSM table initialization will happen in 64bit inside the physics, whereas it is either 64bit or 32bit in fv3atm, depending on whether 32BIT=ON or OFF. Makes sense?

Reply:
Ok. That's probably explained in step 2 (in issue #214), and I was confused as this is step 3.